content/webapp/app.ts

```diff
@@ -124,15 +124,11 @@ const appPromise = nextApp
 
     router.get('/preview', async ctx => {
       // Kill any cookie we had set, as it think it is causing issues.
-      ctx.cookies.set(Prismic.previewCookie);
+      ctx.cookies.set('io.prismic.preview');
```

Comment:
should this be [`prismic.cookie.preview`](https://prismic.io/docs/technologies/prismic-client-v6-migration-guide#replace-previewcookie-with-cookie.preview)?

Reply:
Hmm. I got this value by logging the value of `Prismic.previewCookie` in a local build. Let's unset both of them, just to be safe.